app/qml/ManageInvitationsPage.qml

```diff
@@ -107,6 +107,7 @@ Page {
           model: InvitationsProxyModel {
             invitationsSourceModel: InvitationsModel {
               merginApi: __merginApi
+              fetchFromServer: false
```

Comment:
If you set `fetchFromServer` to false here, it will never call the `v1/invitations`, will it? My understanding is that you need to make a property in `ManageInvitationsPage` and change this flag based on that property

Reply:
Right, we don't want to query server again, as invitations already fetched as a part of the user info request.

Reply:
Oh, right. I've forgotten that we use a different component on the `switchWorkspace` page. My bad!